packages/react-component-library/src/components/Timeline/Timeline.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
```

Comment:
It feels like there could be a `Timeline.test.tsx` covering a bunch of scenarios here.